### PR TITLE
ui/project view

### PR DESCRIPTION
**Pull request by nichanank**
- UI improvements to home page, project page, modal
- add validation to modal buttons so that the `approve` steps are reflected in the UI

### CODE REVIEW

File: client/packages/react-app/src/pages/Projects/index.js [hunk 1]
```diff
@@ -1,51 +1,144 @@
...
+            }}>View</Button>
+          </CardColumn>
+        </CardRow>
         {projectDetailsModalIsOpen ?
               <ProjectDetailsModal 
-              recipient={selectedProjectRecipient}
-              salary={selectedProjectSalary.toString()}
-              startedSurplus={selectedProjectStartedSurplus}
-              principal={selectedProjectPrincipal.toString()}
+              bTokenBalance={ethers.utils.formatEther(selectedProjectBTokenBalance)}
```
Comment (Pongch) [inline client/packages/react-app/src/pages/Projects/index.js:134]: 👌 formatting to Ether unit

File: client/packages/react-app/src/pages/Projects/index.js [hunk 2]
```diff
@@ -1,51 +1,144 @@
...
+              salary={ethers.utils.formatEther(selectedProjectSalary)}
```
Comment (Pongch) [inline client/packages/react-app/src/pages/Projects/index.js:135]: I forgot how React works, but essentially `setSelectedProjectSalary ` would automatically set the value for `selectedProjectSalary`, correct?

**Review by Pongch — APPROVED** [review]
lgtm!

File: client/packages/react-app/src/pages/Projects/index.js (hunk 2 above)
Comment (nichanank) [reply client/packages/react-app/src/pages/Projects/index.js:135]: yep and we are passing in that value for the modal component to display

**Pull request merged**